app/src/main/java/com/example/recipeopedia/fragments/RecipeListFragment.java

```diff
@@ -49,15 +49,17 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-
         mRecipes = new ArrayList<>();
         rvRecipes = view.findViewById(R.id.rvRecipes);
         recipeAdapter = new RecipeAdapter(mRecipes);
         rvRecipes.setAdapter(recipeAdapter);
         rvRecipes.setLayoutManager(new LinearLayoutManager(getContext()));
 
+        queryRecipes("chicken"); // keyword is CHICKEN to not overload API with so many requests
+
         EditText etSearch = view.findViewById(R.id.etSearch);
-        etSearch.addTextChangedListener(new TextWatcher() {
+        // TODO: uncomment below for final product
+        /*etSearch.addTextChangedListener(new TextWatcher() {
```

Comment:
```suggestion
        /*etSearch.addTextChangedListener(new TextWatcher()) {
```

Reply:
I know this is commented out for now, but I noticed you're missing a closing bracket

Reply:
it's an anonymous function so i don't need the closing bracket there!